src/libPMacc/include/cuSTL/cursor/SafeCursor.hpp

```diff
@@ -128,7 +128,9 @@ class SafeCursor : public Cursor
     HDINLINE void checkValidity() const
     {
         if(!this->enabled) return;
+#ifndef _MSC_VER
```

Comment:
Oh, yes that is true. nvcc supports this in device code even with MSVC.
This was a hack required by alpaka because MSVC does not support this in host code.
This will be removed from this PR.